modules/aws_asg/kill-old-containers.bash

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+set -e
+set -o pipefail
+
+__die() {
+  local status="${1}"
+  local code="${2}"
+  local count="${3}"
+  logger "time=$(date -u +%Y%m%dT%H%M%S) " \
+    "prog=$(basename "${0}") status=${status} count=${count}"
+  exit "${code}"
+}
+
+main() {
+  worker_cid=$(docker inspect travis-worker --format '{{ .Id }}')
+  cids=$(docker ps -q --filter before="$worker_cid")
```

Comment:
nitpick: `worker_cid` and `cids` should both be declared as locals prior to subshell definition